core/types/src/attestation_key.rs

```diff
@@ -39,25 +38,25 @@ new_type_accessors_impls! {
     Id, u16;
 }
 
-/// The type of quote signature.  Only valid for EPID quotes.
+/// The type of quote.  Only valid for EPID quotes.
 #[derive(Debug, Clone, Hash, PartialEq, Eq)]
 #[non_exhaustive]
 #[repr(u16)]
-enum QuoteSignature {
+pub enum QuoteSignatureType {
```

Comment:
Rusty name for this would be `QuoteSignatureKind`.

Reply:
renamed to `QuoteSignatureKind`